ignite/cmd/network_request_add_account.go

```diff
@@ -0,0 +1,92 @@
+package ignitecmd
+
+import (
+	"errors"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/ignite/cli/ignite/services/network"
+	"github.com/ignite/cli/ignite/services/network/networkchain"
+)
+
+// NewNetworkRequestAddAccount creates a new command to send add account request
+func NewNetworkRequestAddAccount() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "add-account [launch-id] [address] [coins]",
```

Comment:
`launch-id` and `address` aren't optional
```suggestion
		Use:   "add-account launch-id address [coins]",
```